book/4 - MultiWorlds/1 - Multiworlds Overview.html

```diff
@@ -0,0 +1,21 @@
+<h4>Multiple worlds</h4>
+
+<p>
+	Cuberite supports multiple worlds. Each world has its own `world.ini` file. Additional Worlds can be added by editing <code>server.ini</code>. See the example below.
```

Comment:
Any pointers? I can't think of a better way to explain that. Sometimes an example is the simplest.